awscli/topics/config-vars.rst

```diff
@@ -60,29 +60,31 @@ General Options
-max_attempts         N/A         max_attempts          AWS_MAX_ATTEMPTS      Number of total requests
--------------------- ----------- --------------------- --------------------- ----------------------------
-retry_mode           N/A         retry_mode            AWS_RETRY_MODE        Type of retries performed
-==================== =========== ===================== ===================== ============================
+==================== ============== ===================== ===================== ============================
+Variable             Option         Config Entry          Environment Variable  Description
+==================== ============== ===================== ===================== ============================
+profile              --profile      N/A                   AWS_PROFILE           Default profile name
+-------------------- -------------- --------------------- --------------------- ----------------------------
+region               --region       region                AWS_DEFAULT_REGION    Default AWS Region
+-------------------- -------------- --------------------- --------------------- ----------------------------
+output               --output       output                AWS_DEFAULT_OUTPUT    Default output style
+-------------------- -------------- --------------------- --------------------- ----------------------------
+cli_timestamp_format N/A            cli_timestamp_format  N/A                   Output format of timestamps
+-------------------- -------------- --------------------- --------------------- ----------------------------
+cli_follow_urlparam  N/A            cli_follow_urlparam   N/A                   Fetch URL url parameters
+-------------------- -------------- --------------------- --------------------- ----------------------------
+ca_bundle            --ca-bundle    ca_bundle             AWS_CA_BUNDLE         CA Certificate Bundle
+-------------------- -------------- --------------------- --------------------- ----------------------------
+parameter_validation N/A            parameter_validation  N/A                   Toggles parameter validation
+-------------------- -------------- --------------------- --------------------- ----------------------------
+tcp_keepalive        N/A            tcp_keepalive         N/A                   Toggles TCP Keep-Alive
+-------------------- -------------- --------------------- --------------------- ----------------------------
+max_attempts         N/A            max_attempts          AWS_MAX_ATTEMPTS      Number of total requests
+-------------------- -------------- --------------------- --------------------- ----------------------------
+retry_mode           N/A            retry_mode            AWS_RETRY_MODE        Type of retries performed
+-------------------- -------------- --------------------- --------------------- ----------------------------
+N/A                  --no-cli-pager N/A                   N/A                   Disable output to pager
```

Comment:
For the columns, I think we should fill this out with the other configuration options so for the first column it would be `cli_pager`, the third `cli_pager`, and the fourth `AWS_PAGER`. Then for the description we can set it as `Redirect output to a pager`

Reply:
I was thinking about it. But `--no-cli-pager` works slightly different than the rest it doesn't `Redirect output to a pager`

Reply:
How about `Redirect/Disable output to a pager` then? This table is used for more general configuration options so if the pager functionality is included in this table it should included the config variable and environment variable.